mlrun/package/utils/_archiver.py

```diff
@@ -18,6 +18,8 @@
 from abc import ABC, abstractmethod
 from pathlib import Path
 
+import mlrun.utils
```

Comment:
I don't get why you need to import `mlrun.utils` just for `datetime.now()`, the `mlrun.utils.now_date()` is kinda redundant in my opinion.

Reply:
well it gives something (timezone aware)